install/pre-install-payload/payload.sh

```diff
@@ -77,21 +78,21 @@ function build_payload() {
 		docker push "${registry}:${kernel_arch}-${tag}"
 	done
 
-	purge_previous_manifests ${registry}:${tag}
-	purge_previous_manifests ${registry}:latest
+	purge_previous_manifests "${registry}":"${tag}"
```

Comment:
I'd be perhaps more human-readable to use a single quotation for the whole string `purge_previous_manifests "${registry}:${tag}"` (everywhere where it applies)